examples/ovs-cni.yml

```diff
@@ -51,6 +51,10 @@ spec:
         volumeMounts:
           - name: ovs-var-run
             mountPath: /host/var/run/openvswitch
+        resources:
+          requests:
+            cpu: "5m"
```

Comment:
This does not look right. With 5m CPU, it must be way too slow to report anything. Have you tried setting the limit to 5m to see how it behaves on the minimum?

Reply:
Actual measured numbers

```
create / delete 1000 ovs-bridges |  6m / 11Mi
Suggested |  5m / 10Mi
```

And its the minimum not the maximum
If we set it to 5m it will just simulate a really really corner case where it just get what it guarrnteed
but it can try

Reply:
For the record, copying from another thread:

I know it is just a request. But it is not more useful than 0m if it does not guarantee the application being useful, and that would defy the purpose of this effort.

Setting the request value as limit is a simple way to confirm that the application will be able to serve even under the tight resource situation.

Reply:
Well with 6m we managed to server 1000 bridges, this is where the numbers came from, but np i will try

Reply:
limited to 5m CPU, 
create  / delete of 1000 ovs bridges reflected on the node yaml (and even pretty fast comparing to when the command finished)

Reply:
Awesome, sounds good then

Reply:
let me please more accurate with the comment (wanted to EDIT but wasn't fast enough)

It does take several seconds afterwards
(the original command is long, and then after it finishes it takes lets say roughly 20 - 40 seconds to finish)

Reply:
Not sure I understand. Once all the bridges are created, it takes 20 - 40 seconds to report them as resources? Did you count in the polling interval? How fast is it without the limit?

Reply:
20 - 40 second is just the extra time, until it finish report them all
lets say that you create 1000 bridges, 
when the command itself finished and the 1000 bridges are there, 800 are already reported,
just the 200 left takes more time after the original command finished

didn't count in the polling interval nor have the info without the limit.

if we do care for it still please let me know 
I think we are safe enough, just wanted to be more accurate